aptos-move/aptos-vm/src/block_executor/mod.rs

```diff
@@ -178,6 +178,49 @@ impl BlockAptosVM {
         concurrency_level: usize,
         maybe_block_gas_limit: Option<u64>,
         transaction_commit_listener: Option<L>,
+    ) -> Result<Vec<TransactionOutput>, VMStatus> {
+        Self::run_block(
+            executor_thread_pool,
+            signature_verified_block,
+            state_view,
+            concurrency_level,
+            maybe_block_gas_limit,
+            transaction_commit_listener,
+            true,
```

Comment:
execute block passes true to variable called is_simulation, I think?

Reply:
I remember I had that, thought I did change it🙃